src/BackTranslationHelper/BackTranslationHelperCtrl.cs

```diff
@@ -535,13 +586,23 @@ private void AddEncConverterToolStripMenuItem_Click(object sender, EventArgs e)
             }
 
             System.Diagnostics.Debug.Assert(_model != null);
-            Initialize(!String.IsNullOrEmpty(_model.TargetDataPreExisting));
+            Initialize(!String.IsNullOrEmpty(_model.TargetDataPreExisting) && !hideCurrentTargetTextToolStripMenuItem.Checked);
             UpdateData(_model);
         }
 
 		private void UpdateEditableTextBox(TextBox textBoxFrom)
 		{
+			if (IsModified)
+			{
+				var res = MessageBox.Show($"Do you want to overwrite the current changes in the 'Target Translation' box?", BackTranslationHelperDataSource.ProjectName, MessageBoxButtons.YesNoCancel);
```

Comment:
If there isn't any difference between no and cancel, this could just be MessageBoxButtons.YesNo

Reply:
changed to YesNo